### PR TITLE
Spanish by subtle

### PR DESCRIPTION
**Pull request by 0x5ubt13**
# Description
- Completed the Spanish translation in `app/src/main/res/values-es`

See #172

### CODE REVIEW

**Review by johnjohndoe — CHANGES_REQUESTED** [review]
Thank you for the Spanish update. I took a first look. There a few things to correct. Please be welcome to rewrite/modify your existing commits and `git push --force` your `spanish_by_subtle` branch to update this pull request. If you need help let me know.

**Comment by johnjohndoe** [comment]
@0x5ubt13 Please let me know if I can help with the pull request - but take your time if you are busy these days.

**Comment by johnjohndoe** [comment]
@0x5ubt13 If you do not have time to finish this, please let me know so I can offer the issue to somebody else.

**Comment by 0x5ubt13** [comment]
Oh I forgot about this project completely, I'm so sorry about it. I'll take a look at the corrections tonight and get it done

**Review by 0x5ubt13 — COMMENTED** [review]
I've reviewed all the comments and applied the necessary changes as per their request.

**Comment by 0x5ubt13** [comment]
All changes to the pull request should make the whole lot hopefully meet your criteria :)

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -108,6 +107,19 @@
...
+    <string name="placeholder_session_day">"          Hoy"</string>
+    <string name="validation_error_invalid_url">"         URL <xliff:g example="Engelsystem" id="urlType">%s</xliff:g> inválida."</string>
+    <string name="validation_error_url_with_incomplete_query">"         <xliff:g example="Engelsystem" id="urlType">%s</xliff:g> URL con parte de la query incompleta.     "</string>
+    <string name="preference_summary_engelsystem_json_export_url">"         <![CDATA[Para ver tus turnos por favor pon tu URL <u>personal</u> aquí.<br/><br/>         1. Haz login en tu cuenta Engelsystem.<br/>         2. Copia la URL detrás del botón de <u>Exportación en JSON</u> en tu página de perfil.<br/>         3. Pega la URL aquí.<br/><br/>         Los horarios antes y después de los días principales de la conferencia <u>no</u> serán mostrados.]]>"</string>
+    <string name="validation_error_url_without_api_key">"         <xliff:g example="Engelsystem" id="urlType">%s</xliff:g> URL sin API key.     "</string>
+    <string name="validation_error_url_without_query">"         <xliff:g example="Engelsystem" id="urlType">%s</xliff:g> URL sin query.     "</string>
```
[the remark this answers is not in the document]
Comment (johnjohndoe) [reply app/src/main/res/values-es/strings.xml]: This was not fixed. I took care of it.

File: app/src/main/res/values-es/strings.xml [hunk 2]
```diff
@@ -108,6 +107,19 @@
...
+    <string name="engelsystem_shifts_parsing_error_forbidden">API key inválida para turnos de Engelsystem</string>
+    <string name="engelsystem_shifts_parsing_error_not_found">URL inválida para turnos de Engelsystem</string>
+    <string name="placeholder_session_day">"          Hoy"</string>
+    <string name="validation_error_invalid_url">"         URL <xliff:g example="Engelsystem" id="urlType">%s</xliff:g> inválida."</string>
+    <string name="validation_error_url_with_incomplete_query">"         <xliff:g example="Engelsystem" id="urlType">%s</xliff:g> URL con parte de la query incompleta.     "</string>
+    <string name="preference_summary_engelsystem_json_export_url">"         <![CDATA[Para ver tus turnos por favor pon tu URL <u>personal</u> aquí.<br/><br/>         1. Haz login en tu cuenta Engelsystem.<br/>         2. Copia la URL detrás del botón de <u>Exportación en JSON</u> en tu página de perfil.<br/>         3. Pega la URL aquí.<br/><br/>         Los horarios antes y después de los días principales de la conferencia <u>no</u> serán mostrados.]]>"</string>
```
[the remark this answers is not in the document]
Comment (johnjohndoe) [reply app/src/main/res/values-es/strings.xml]: You did not move the string to another file. I did that now.

**Review by johnjohndoe — APPROVED** [review]
Thank you @0x5ubt13 for the "late night" update ;-)

I had to fix a few left-overs (I already commented those). Also, you deleted the `session_duration` string which I brought back. Finally, I squashed your commits into a single commit. As a side note for future commits: please avoid merging `master` into your branch. Instead you can rebase your branch.

Now, the translation looks fine. I checked in on a Pixel 2, Android 10. Thank you once again for your time!

**Comment by johnjohndoe** [comment]
Happy new year @0x5ubt13! I started managing translations for the EventFahrplan at [Crowdin](https://crowdin.com/project/eventfahrplan). I like to invite you to join there if you like.

**Pull request merged**